pom.xml

```diff
@@ -686,9 +703,9 @@ THE SOFTWARE.
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-      <version>3.12.0</version>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>commons-lang3-api</artifactId>
+      <version>3.12.0-36.vd97de6465d5b_</version>
```

Comment:
Isn't the version already set by the BOM?
```suggestion
```

Reply:
no